### PR TITLE
LFS Alpha fix-ups

### PR DESCRIPTION
**Pull request by davecharles**
### What is the context of this PR?
Social Demographic submitted comprehensive comments on the first cut alpha for LFS; this PR implements many of their change requests and also incorporates some fixes.

### How to review 
Run local eq-survey-runner instance using this branch and local go-launcher (LFS already in go-launcher master), or use this [quick launch URL](https://preprod-surveys-launch.eq.ons.digital/quick-launch?url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-2072-lfs-alpha-fixups/data/en/labour_force.json).

**Note** there are some oddities and inconsistencies with other eq schemas at the request of SD - any queries please come and look at marked up hard copies I have.

### CODE REVIEW

File: data/en/labour_force.json [hunk 1]
```diff
@@ -26,28 +26,126 @@
...
+                                        "block": "household-composition",
+                                        "when": [{
+                                            "id": "address-check-answer",
+                                            "condition": "not set"
+                                        }]
+                                    }
+                                },
+                                {
+                                    "goto": {
+                                        "block": "address-type-check-block"
```
Comment (shahi645) [inline data/en/labour_force.json:82]: @davecharles This routing rule won't ever be hit? unless its no in radio options?

**Comment by shahi645** [comment]
@davecharles good to approve this, once it passes schema-validator

**Review by nealedj — APPROVED** [review]

File: data/en/labour_force.json (hunk 1 above)
Comment (davecharles) [reply data/en/labour_force.json:82]: The intention is, if response is no - route to `address-type-check-block`

**Review by shahi645 — APPROVED** [review]

**Comment by davecharles** [comment]
Added don't merge label - I'm finalising the latest (definitely last) SD requests and when ready will up-merge master updates for date formatting.

**Pull request merged**